src/main/java/group1/mips_simulator/components/cpuParts/RegisterFile.java

```diff
@@ -63,4 +63,21 @@ public Register getIXR(int i){
     public Register getPC(){
         return this.PC;
     }
+    public Register getCC(){
+        return this.CC;
+    }
+    public Register getIR(){
+        return this.IR;
+    }
+    public Register getMAR(){
+        return this.MAR;
+    }
+    public Register getMBR(){
+        return this.MBR;
+    }
+    public Register getMFR(){
+        return this.MFR;
+    }
+//    public Register getFPR() { return this.FPR; }
```

Comment:
Should this be removed?

Reply:
MFR or FPR? sorry still getting used to githubs review thing and its highlighted all these lines

im guessing not MFR,  since we are definitely supposed to have a Machine Fault Register, unless you mean it should be kept somewhere else?
if FPR, leave it in there for when we do need to implement float point, this is a placeholder for part 4 of the project

Reply:
Oh, sorry line 81 the comment. Should it be removed? If you wanna keep it then please add a todo statement explaining what needs to happen here in order to un-comment it.

Reply:
When I see comments like this in the code (without a TODO statement) I just automatically remove them. Because I assume it's some scratch work that someone forgot to remove. So if you wanna keep it, then protect it with a todo statement.

Reply:
i getchu, fair enough